packages/fast-usdc/src/exos/advancer.js

```diff
@@ -271,26 +271,26 @@ export const prepareAdvancerKit = (
          * @param {AdvancerVowCtx} ctx
          */
         onFulfilled(result, ctx) {
-          const { notifyFacet } = this.state;
+          const { notifier } = this.state;
           const { advanceAmount, destination, ...detail } = ctx;
           log('Advance succeeded', { advanceAmount, destination });
           // During development, due to a bug, this call threw.
           // The failure was silent (no diagnostics) due to:
           //  - #10576 Vows do not report unhandled rejections
           // For now, the advancer kit relies on consistency between
-          // notifyFacet, statusManager, and callers of handleTransactionEvent().
+          // notify, statusManager, and callers of handleTransactionEvent().
```

Comment:
find-replace error. Not sure it's worth another round through the CI gauntlet but if I do another push I'll include this.